commands/defensestate.py

```diff
@@ -0,0 +1,25 @@
+from commands2 import CommandBase
+from subsystems.drivesubsystem import DriveSubsystem, SwerveModule
+from wpimath.geometry import Rotation2d
+
+
+class DefenseState(CommandBase):
+    def __init__(self, drive: DriveSubsystem) -> None:
+        CommandBase.__init__(self)
+        self.setName(__class__.__name__)
+        self.drive = drive
+
+        self.addRequirements([self.drive])
+
+    def initialize(self) -> None:
+        print(f"Command: {self.getName()}")
```

Comment:
Should add an `def end(self):` to the command to print the corresponding "DONE" message so that we can check when the command was finished from the logs